src/main/java/jacchm/footballapp/mapping/dto/StandingDTO.java

```diff
@@ -7,12 +8,28 @@
 @Data
 public class StandingDTO {
 
-    private Integer id;
+    private Integer id = null;

```

Comment:
To chyba jest redundantne - id z automatu jest nullem chyba przy utworzeniu obiektu.